jmbitcoin/jmbitcoin/__init__.py

```diff
@@ -2,7 +2,17 @@
 from jmbitcoin.secp256k1_main import *
 from jmbitcoin.secp256k1_transaction import *
 from jmbitcoin.secp256k1_deterministic import *
-from jmbitcoin.btscript import *
-from jmbitcoin.bech32 import *
+from jmbitcoin.snicker import *
 from jmbitcoin.amount import *
+from bitcointx import select_chain_params
+from bitcointx.core import (x, b2x, b2lx, lx, COutPoint, CTxOut, CTxIn,
+                            CTxInWitness, CTxWitness, CTransaction,
+                            CMutableTransaction, Hash160,
+                            coins_to_satoshi, satoshi_to_coins)
+from bitcointx.core.key import KeyStore
+from bitcointx.core.script import (CScript, OP_0, SignatureHash, SIGHASH_ALL,
+                                   SIGVERSION_WITNESS_V0, CScriptWitness)
+from bitcointx.wallet import (CBitcoinSecret, P2WPKHBitcoinAddress, CCoinAddress,
```

Comment:
Note: `CBitcoinSecret` is a backwards-compatibility alias, and is defined only for mainnet. The 'native' class for keys in bitcointx is `CCoinKey` (chain-unsepcific, will work if switched to testnet, regtest), `CBitcoinKey` - explicitly for mainnet.

`P2WPKHCoinAddress` is also chain-unspecific version of `P2WPKHBitcoinAddress` which is mainnet-only

Reply:
`CBitcoinSecret` this is an erroneous import, it'll be caught by `flake` when I get round to going through all the imports. I suspect there will be a few of these.

> P2WPKHCoinAddress is also chain-unspecific version of P2WPKHBitcoinAddress (which is mainnet-only)

Oh; I didn't realise the mainnet-only part. The code in secp256k1_transaction.py is using this class but only transiently so I didn't experience that error. But this will be fixed, thanks for the spot.

Reply:
Addressed both points in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536/commits/1e2fc3848295a05f4ee685fd435f277c378acadc